### PR TITLE
add some tests to kiwidrive

### PR DESCRIPTION
**Pull request by ariovistus**
fix some bugs

### CODE REVIEW

**Comment by ariovistus** [comment]
I feel that the arm and winch code and everything not pertaining to the kiwidrive ought to be in robot.py, not kiwi.py.

**Comment by bb010g** [comment]
Why not separate files, like arm.py & winch.py?

File: kiwidrive/xbox.py [hunk 1]
```diff
@@ -34,8 +34,8 @@ def analog_rot(self):
                self.joystick.getRawButton(XboxController.BUTTON_BUMP_LEFT)
 
     def analog_winch(self):
-        return self.joystick.getRawJoy(XboxController.JOY_TRIG_R) -\
-               self.joystick.getRawJoy(XboxController.JOY_TRIG_L)
+        return self.joystick.getRawButton(XboxController.JOY_TRIG_R) -\
+               self.joystick.getRawButton(XboxController.JOY_TRIG_L)
```
Comment (bb010g) [inline kiwidrive/xbox.py]: ``JOY_TRIG_{R,L}`` is a Joy, not a Button. Did you want to put in ``BUTTON_BUMP_{R,L}``?

Comment (ariovistus) [inline kiwidrive/xbox.py]: getRawJoy is not a function. looks like it should be getRawAxis

Comment (bb010g) [inline kiwidrive/xbox.py]: :worried: Yes.

**Comment by ariovistus** [comment]
for 10 lines of code, separate files seems a bit overkill

**Pull request merged**